share/config.py

```diff
@@ -290,6 +290,11 @@ def add_output(self, output_type: str, **kwargs: Any) -> None:
 
         output: Optional[Output] = None
         if output_type == "elasticsearch":
+            if "es_index_or_datastream_name" in kwargs:
+                if "datastream" not in kwargs:
+                    kwargs["datastream"] = kwargs["es_index_or_datastream_name"]
```

Comment:
Hmm in Firehose integration, we changed the config parameter to `es_datastream_name` instead. Not sure if we want to match it here?